src/transformers/training_args.py

```diff
@@ -696,6 +699,11 @@ class TrainingArguments:
             for instruction fine-tuning. Check out the [original paper](https://arxiv.org/abs/2310.05914) and the
             [original code](https://github.com/neelsjain/NEFTune). Support transformers `PreTrainedModel` and also
             `PeftModel` from peft.
+        galore_target_modules (`List[str]`, *optional*):
```

Comment:
It would be cool if we could use the same mechanism as for `target_modules` in PEFT:

- allow str for regex match
- if list of str, not only match exact names, but also if the module name ends with the target module name
- passing `all-linear` to match all linear layers

But I understand that we don't want to copy the whole mechanism to transformers. Maybe we can think of a way to factor this code out in the future, so that we can re-use it in multiple places.

Reply:
This should be now supported !